src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -668,7 +682,7 @@ public function getChanger()
      */
     public function getCreated()
     {
-        return $this->getFileVersion()->getCreated();
```

Comment:
we should use the same method to get the entity in all functions here... sometimes it uses `$this->getFileVersion()` and the others uses `$this->entity` could you change it to the same in this PR?